test/recipe/admin/components/Baked/Menu.vue

```diff
@@ -1,8 +1,9 @@
 <template>
-  <Menubar :model="data" />
+  <Menubar v-if="schema" :model="schema.items" />
```

Comment:
menu should use data, not schema. no need to change the component to achieve the intended result